### ISSUE TITLE
NSDebugDescriptionErrorKey does not change the error description on the Issues page

### ISSUE THREAD

**Issue by imjamesim** [getsentry/sentry-docs#5540]
### Core or SDK?

Platform/SDK

### Which part? Which one?

ios/usage/#customizing-error-descriptions

### Description

Excerpt in Customizing Error Descriptions section on the Usage for iOS page:

> By default, Sentry will display the error code as the error description. For custom error types, you may want to provide a custom description that makes it easier to identify the error in the Issues page. You can provide a custom description for NSError values by adding a description to the userInfo dictionary with the key NSDebugDescriptionErrorKey.

Although I've added a description to the userInfo dictionary with `NSDebugDescriptionErrorKey`, I'm not seeing the description anywhere on the Issues page. I do, however, see the description under the User Info section once I click into the specific issue's details page.

### Suggested Solution

Show the custom descriptions on the Issues page.

**Comment by imatwawana** [comment]
@getsentry/team-mobile - should the text here be changed to say the Issue Details page?

**Comment by imjamesim** [comment]
If at all possible, I would like to request Sentry to show the custom description instead of the error code on the Issues page. Sadly, the error code can sometimes be completely meaningless, in which case it becomes very difficult to distinguish issues from one another.

<img width="440" alt="Screen Shot 2022-09-23 at 8 41 22 PM" src="https://user-images.githubusercontent.com/28934835/192078594-e0557fde-2696-4ddd-8f76-5e7c28f09942.png">

**Comment by brustolin** [comment]
@imatwawana I dont think this is a documentation problem but an implementation one. I'll transfer this to coca repo.
